extensions/warp-mp-ipfs/src/store/friends.rs

```diff
@@ -155,7 +156,7 @@ impl<T: IpfsTypes> FriendsStore<T> {
         let store = Self {
             ipfs,
             identity,
-            did_key,
+            did_key: did_key.clone(),
```

Comment:
cloning shouldnt be when you can use it from `store.did_key`

Reply:
ok, done